src/spaces/xyz-d65.js

```diff
@@ -10,8 +10,9 @@ export default new ColorSpace({
 	},
 	white: "D65",
 	formats: {
-		color: {}
+		color: {
+			ids: ['xyz'],
```

Comment:
We should probably add both, for clarity

Reply:
Done!